src/providers/fireworks-ai/index.ts

```diff
@@ -0,0 +1,36 @@
+import { ProviderConfigs } from '../types';
+import FireworksAIAPIConfig from './api';
+import {
+  FireworksAIChatCompleteConfig,
+  FireworksAIChatCompleteResponseTransform,
+  FireworksAIChatCompleteStreamChunkTransform,
+} from './chatComplete';
+import {
+  FireworksAICompleteConfig,
+  FireworksAICompleteResponseTransform,
+} from './complete';
+import {
+  FireworksAIEmbedConfig,
+  FireworksAIEmbedResponseTransform,
+} from './embed';
+import {
+  FireworksAIImageGenerateConfig,
+  FireworksAIImageGenerateResponseTransform,
+} from './imageGenerate';
+
+const FireworksAIConfig: ProviderConfigs = {
+  complete: FireworksAICompleteConfig,
+  chatComplete: FireworksAIChatCompleteConfig,
+  embed: FireworksAIEmbedConfig,
+  imageGenerate: FireworksAIImageGenerateConfig,
+  api: FireworksAIAPIConfig,
+  responseTransforms: {
+    complete: FireworksAICompleteResponseTransform,
```

Comment:
stream-complete transformer function is present but it not mapped here. Can you please check once?

Reply:
done